client/jetpack-onboarding/summary-next-steps.jsx

```diff
@@ -16,12 +16,14 @@ import { getEditorNewPostPath } from 'state/ui/editor/selectors';
 import { getJetpackOnboardingSettings } from 'state/selectors';
 import { isJetpackSite } from 'state/sites/selectors';
 
-const NextSteps = ( { siteId, steps } ) => (
+const NextSteps = ( { handleNextStepSelection, siteId, steps } ) => (
```

Comment:
Do we want to have a default `noop` value for `handleNextStepSelection`? Right now if you don't pass a `handleNextStepSelection` prop to `NextSteps`, it'll blow up.